gpiozero/notes_data.py

```diff
@@ -1,7 +1,4 @@
 notes = {
-    """
```

Comment:
Or you could have moved the comments above the `notes = {` line, instead of deleting them entirely :wink:

Reply:
I didn't think they were necessary :P